test/support/utils_test_base.h

```diff
@@ -0,0 +1,698 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+#ifndef _UTILS_TEST_BASE_H
+#define _UTILS_TEST_BASE_H
+
+#include <memory>
+#include <vector>
+#include <initializer_list>
+#include <utility>
+
+#include "utils_const.h"
+#include "utils_sequence.h"
+#include "utils_invoke.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#include "sycl_alloc_utils.h"
+#endif // TEST_DPCPP_BACKEND_PRESENT
+
+namespace TestUtils
+{
+////////////////////////////////////////////////////////////////////////////////
+/// enum UDTKind - describe test source data kinds
+enum class UDTKind
+{
+    eKeys = 0,  // Keys
```

Comment:
comment should be changed: "ID for the first input data"

Reply:
Done